ckanext/packagezip/helpers.py

```diff
@@ -8,5 +8,10 @@ def has_packagezip(pkg):
 def packagezip_url(pkg):
     return toolkit.url_for('zipfile', name=pkg.name)
 
+def packagezip_size(pkg):
```

Comment:
Are you confident no return value is safe?  No context for me to judge. Presumably there's a None check elsewhere?